app/assets/stylesheets/application.scss

```diff
@@ -12,29 +21,152 @@
   display: none;
 }
 
-.input_checkbox input {
-  width: auto !important;
+.div-form {
+  padding-left: 1em;
+  padding-bottom: 1em;
+}
+
+.div-elem, 
+.form-elem {
+  padding: 1.5em 0;
+}
+
+.elem-name {
+  padding-right: 1em;
+  font-size: 130%;
```

Comment:
Personally I try to use `rem` as the unit for fonts but I recently heard someone suggest using `vw` or `vh` as it'll scale in real-time with the browser, which is pretty cool.